steps/rhelah.py

```diff
@@ -227,6 +236,50 @@ def step_impl(context):
     assert rollback_result, "Error while running 'atomic host rollback'"
 
 
+@when('atomic stop container')
+def step_impl(context):
+    '''stop running container'''
+    container_id = get_running_container_id(context)
+    assert container_id, "No running container"
+    assert context.remote_cmd('command',
+                               remote_user='root',
```

Comment:
@miabbott, you're right, it's not necessary in here, I will remote it from codes.